sellercenter-product-port/src/main/java/com/gfg/sellercenter/product/port/Product.java

```diff
@@ -0,0 +1,88 @@
+package com.gfg.sellercenter.product.port;
+
+import lombok.Data;
+import lombok.AllArgsConstructor;
+
+import javax.validation.constraints.NotEmpty;
+import javax.validation.constraints.NotNull;
+import java.time.ZonedDateTime;
+
+@Data
+@AllArgsConstructor
+public class Product {
+    @NotNull
+    private final Integer id;
+
+    private final Integer sourceId;
+
+    private final String sku;
+
+    @NotNull
+    private final Price price;
+
+    @NotNull
+    @NotEmpty
+    private final String sellerSku;
+
+    @NotNull
+    @NotEmpty
+    private final String name;
+
+    private final Double volumetricWeight;
+
+    public boolean hasSpecialPrice() {
+        return price.getSpecialPrice() != null;
```

Comment:
Ehm, I believe it should also check if there is a special price, but expired?

Reply:
Makes sense. Changes are pushed.